src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--start-day--sun calendar--month-length--29">
+      <div class="div calendar__elem"></div>
```

Comment:
```suggestion
      <div class="calendar__elem"></div>
```
Why do you need class div? It's not according to BEM and you don't even use this class